doc/source/user_guide/troubleshoot.rst

```diff
@@ -268,10 +256,7 @@ On CentOS 7, you can install missing dependencies with:
     yum install openssl openssh-clients mesa-libGL mesa-libGLU motif libgfortran
 
 
-.. _installing_mapdl_on_ubuntu:
-
-Ubuntu
-------
+**Ubuntu**
```

Comment:
Why this change?

Reply:
This change was to have the `troubleshoot.rst` better organized.